docs/product-user-guide/src/main/asciidoc/deployment-unit-create-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='deployment-unit-create-proc_{context}']
+= Creating a deployment unit
+
+You can create a deployment unit from a project that was previously built in {CENTRAL}.
+
+.Procedure
+
+. In {CENTRAL}, go to *Menu* -> *Deploy* -> *Execution servers*.
+. Under *Server Configurations*, select an existing configuration or click *New Server Configuration* to create a new configuration.
+. Under *Deployment Units*, click *Add Deployment Unit*.
+. In the table in the dialog, select a GAV and click the *Select* button next to the GAV to populate the deployment unit data fields.
```

Comment:
IBM SG: dialog box n Use only in technical information for programmers where it is important to distinguish parts of the interface. Do not use “dialog”. For most user information, use “window” when referring to a
graphical interface.

the *Select* button - do you need to specify 'button'? IBM: use only the label,
title, descriptive name, or name that is indicated by the tooltip. However, if an inter-
face contains elements with the same label or very similar labels, include the element
type to ensure clarity. 
Wherever possible, I avoid 'button' but maybe it's necessary here.

Reply:
@emmurphy1, I changed to "window". I too avoid "button" and even find it to be a pet peeve. In this case, there is a "Select" column with individual "Select" buttons next to each GAV in the table. I thought specifying "button" in this case was actually helpful. That said, I think "next to the GAV" is helpful enough so I'll remove "button". Thanks!